docs/serverless/08-02-create-git-function.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Create a Function from Git Repository
+type: Tutorials
+---
+
+This tutorial shows how you can create a simple "Hello World!" Function from Git repository.
```

Comment:
`This tutorial shows how you can create a Function from Git repository.` The function from `examples` repo is a more complex than `Hello World!` :)